From 75f0070e66c212733e9b0d1850e98ff67c88ca64 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:56 -0400
Subject: [redhat] kabi: Adding symbol csum_partial (lib/checksum.c)

Message-id: <8b82ee78bcce86074659806d2c90f2db178db783.1410510773.git.poros@redhat.com>
Patchwork-id: 96053
O-Subject: [PATCH RHEL7.1 056/233] kabi: Adding symbol csum_partial (lib/checksum.c)
Bugzilla: 1071097 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335,1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Through API skb_postpull_rcsum. LLT checksums packets to detect network corruption.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/csum_partial
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/csum_partial

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/csum_partial b/redhat/kabi/kabi-module/kabi_ppc64/csum_partial
new file mode 100644
index 0000000..69f9c92
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/csum_partial
@@ -0,0 +1,2 @@
+#1-
+0xe113bbbc csum_partial vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/csum_partial b/redhat/kabi/kabi-module/kabi_x86_64/csum_partial
new file mode 100644
index 0000000..69f9c92
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/csum_partial
@@ -0,0 +1,2 @@
+#1-
+0xe113bbbc csum_partial vmlinux EXPORT_SYMBOL
-- 
1.7.1